src/components/Footer/index.tsx

```diff
@@ -30,75 +32,89 @@ const cardContents = [
     value: "instagram.com/acm.ucsd",
     icon: InstagramIcon,
   },
+  {
+    title: "LinkedIn",
+    value: "acmurl.com/linkedin",
+    icon: LinkedinIcon,
+  },
+  {
+    title: "YouTube",
+    value: "acmurl.com/youtube",
+    icon: YoutubeIcon,
+  },
 ]
+function isValidEmail(input) {
+  const reg = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+  return reg.test(String(input).toLowerCase())
```

Comment:
Despite it being the standard that emails are case insensitive (if you're curious, [this RFC is fun](https://datatracker.ietf.org/doc/html/rfc5321#section-2.3.11)), they're not always treated as such in practice.

Your regex already handles uppercase letters in email, so you don't have to lower case the email address:
```suggestion
  return reg.test(String(input))
```